api-common/pom.xml

```diff
@@ -11,7 +11,7 @@
   <dependencies>
     <dependency>
       <groupId>io.enmasse</groupId>
-      <artifactId>api-model</artifactId>
+      <artifactIdapi-model</artifactId>
```

Comment:
this is still malformed

Reply:
oops sorry again for the carelessness.  fixed now.